authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,24 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+from django.conf import settings
+from rest_framework import authentication, exceptions
+
+from .models import User
 
 """Configure JWT Here"""
```

Comment:
Delete this comment since you've configured JWT

Reply:
Okay...